support/Environments/minerva_clang.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/env sh
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Instructions to **compile** spectre on Minerva:
+#
+# 1. Clone the spectre repository, if you haven't already. A good place is in
+#    your `/home` directory on Minerva:
+#    ```sh
+#    cd /home/yourname
+#    git clone git@github.com:yourname/spectre.git
+#    ```
+# 2. Set the `$SPECTRE_HOME` environment variable to the location of the spectre
+#    repository, e.g. `/home/yourname/spectre`:
+#    ```sh
+#    export SPECTRE_HOME=path/to/spectre
+#    ```
+# 3. Source this script and setup modules:
+#    ```sh
+#    source path/to/this/script.sh`
+#    spectre_setup_modules
+#    ```
+#    Note: Add steps 2 and 3 to your `.bashrc` file if you don't want to repeat
+#    them every time you log in. The `spectre_setup_modules` function only
+#    adjusts your `MODULEPATH` to make the installed modules visible but doesn't
+#    load any, so it's safe to add to your `.bashrc`.
+# 4. Create a build directory, if you don't have one already. A good place is in
+#    your `/work` directory on Minerva:
+#    ```sh
+#    cd /work/yourname
+#    mkdir spectre-build
+#    ```
+#    Note: Add a timestamp or descriptive labels to the name of the build
+#    directory name, since you may create more build directories later, e.g.
+#    `build_YYYY-MM-DD` or `build-clang-Debug`.
+# 5. Run `cmake` to configure the build directory:
+#    ```sh
+#    cd path/to/build/directory
+#    module purge
+#    spectre_run_cmake
+#    ```
+#    Note: Remember to `module purge` to work in a clean environment, unless you
+#    have reasons not to.
+# 6. Compile! With the build directory configured, this script sourced and
+#    modules set up, you can skip the previous steps from now on.
+#    ```sh
+#    module purge
+#    spectre_load_modules
+#    make -j16 ExportCoordinates1D
```

Comment:
if I read the `lscpu` correctly, thats ALL the cores of the entry node! Andre will not be happy ;) Also, if users change the executable to something memory heavy, this will likely kill the node entirely with the current memory usage we have.

Reply:
I added a first step that suggests to compile on a compute node.